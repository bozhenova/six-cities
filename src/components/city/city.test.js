import React from 'react';
import City from './city';
import renderer from 'react-test-renderer';

describe('City component renders correctly', () => {
  const props = {
    key: 42,
    city: 'Amsterdam',
    changeCity: jest.fn(),
    currentCity: 'Amsterdam'
  };

  it('should render a City component', () => {
    const tree = renderer.create(<City {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
