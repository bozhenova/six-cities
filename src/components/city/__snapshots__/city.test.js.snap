// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`City component renders correctly should render a City component 1`] = `
<li
  className="locations__item"
>
  <a
    className="locations__item-link tabs__item tabs__item--active"
    onClick={[Function]}
    onKeyPress={[Function]}
    tabIndex={0}
  >
    <span>
      Amsterdam
    </span>
  </a>
</li>
`;
